opennms-base-assembly/src/main/filtered/bin/opennms

```diff
@@ -851,8 +851,16 @@ case "$COMMAND" in
 
 	configtest)
 		$opennms_echo "Validating XML files: \c"
-		checkXmlFiles || exit $?
-		echo "PASSED"
+		checkXmlFiles
+		ret=$?
+		case $ret in
+			0)
+				echo "PASSED"
+				;;
+			*)
+				# nothing
```

Comment:
checkXmlFiles() itself is quite verbose when XML errors are identified.  I'm not sure that distinguishing between return values 3 and 6 would give any more meaningful information for the user -- they both mean that there were one or more XML syntax errors.